torch/fx/experimental/proxy_tensor.py

```diff
@@ -705,10 +733,26 @@ def wrap_fake(x):
 
             return x
 
+        def extract_and_fixup_fake_mode_from_inp(x):
+            nonlocal fake_tensor_mode
+            if isinstance(x, FakeTensor) and isinstance(fake_tensor_mode, nullcontext):
+                fake_tensor_mode = x.fake_mode
+                if _allow_non_fake_inputs:
+                    new_fake_mode = FakeTensorMode(
+                        allow_fallback_kernels=x.fake_mode.allow_fallback_kernels,
+                        allow_non_fake_inputs=True,
```

Comment:
This seems highly questionable. The original allocation of FakeTensorMode should have gotten this setup; fixing it up after the fact is something is not supported API.

Reply:
Yeah i just moved it up to dynamo, because only reason it exist is to workaround params and buffers. In export, we don't care about their mutation for now (until Brian's refactor), so i think it is ok to just set it to true in export mode only, and remove it later.